src/app/containers/ChartbeatAnalytics/utils/index.js

```diff
@@ -136,7 +136,7 @@ export const getConfig = ({
     sections,
     uid: chartbeatUID,
     title,
-    virtualReferrer: referrer,
+    virtualReferrer: decodeURIComponent(referrer),
```

Comment:
I was wondering if we have to add `decodeURIComponent` here since the returned `referrer` value is not encoded?

Reply:
@thekp the `referrer` value here will be encoded (for canonical) because the `referrer` is returned from the `getReferrer` utility function which encodes the value.

Reply:
ah okay